src/flatfile.cpp

```diff
@@ -13,3 +14,26 @@ fs::path FlatFileSeq::FileName(const CDiskBlockPos& pos) const
 {
     return m_dir / strprintf("%s%05u.dat", m_prefix, pos.nFile);
 }
+
+FILE* FlatFileSeq::Open(const CDiskBlockPos& pos, bool fReadOnly)
```

Comment:
Commit ad59122

Just asking for a follow up, could this be changed to `std::fstream`?

Reply:
Maybe... but a lot of file utilities like `FileCommit`, `AllocateFileRange`, etc would have to be modified as well. I think the better approach would be to return a `CAutoFile` and define the move constructor/assignment operator properly. And probably move it to a different module, like in util.